middleware/logger.go

```diff
@@ -0,0 +1,21 @@
+package middleware
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/rs/zerolog"
+)
+
+// Make a logging middleware handler with the configured logger injected.
+func MakeLoggingMiddlewareHandler(logger zerolog.Logger) func(http.Handler) http.Handler {
+	return func(next http.Handler) http.Handler {
+		handler := func(w http.ResponseWriter, r *http.Request) {
+			requestLogMessage := fmt.Sprintf("%s %s %s", r.RemoteAddr, r.URL, r.Method)
```

Comment:
Should this have a lightweight message explaining where it is. Something like `Handling Request: %s %s %s`. My thought here is that once there are more logs, it will be unclear what this log is.